processor/src/main/java/org/mapstruct/ap/internal/model/AbstractBaseBuilder.java

```diff
@@ -50,7 +50,27 @@ public B method(Method sourceMethod) {
         return myself;
     }
 
-    boolean isDisableSubMappingMethodsGeneration() {
+    /**
+     * Checks if MapStruct is allowed to generate an automatic sub-mapping between {@code sourceType} and @{code
+     * targetType}.
+     * This will evaluate to {@code true}, when:
+     * <li>
+     * <ul>Automatic sub-mapping methods generation is not disabled</ul>
+     * <ul>MapStruct is allowed to generate an automatic sub-mapping between the {@code sourceType} and {@code
+     * targetType}</ul>
+     * </li>
+     *
+     * @param sourceType the source type that we want to generate a sub-mapping from
```

Comment:
I'm not in favor of using 'we'.. Perhaps: `candidate source type to generate a sub-mapping from`?